app/models/concerns/date_of_birth.rb

```diff
@@ -0,0 +1,22 @@
+module DateOfBirth
```

Comment:
This validation would be better to do as an `ActiveModel::Validator` as it is easier to test and we avoid having a module with a strange name.

The resulting model validation would look something like:

```ruby
validates :date_of_birth, age: true
```